src/frontend/src/flows/registerDisabled.ts

```diff
@@ -1,47 +1,55 @@
 import { html, render } from "lit";
 import { warnBox } from "../components/warnBox";
 
-const pageContent = html`
+const pageContent = (onCancel: () => void) => html`
   <div class="l-container c-card c-card--highlight">
-    <h1>Create a new Internet Identity Anchor</h1>
-    ${warnBox({
-      title: "Registration Disabled",
-      message: html`<p>
-          You are <b>not</b> browsing this website on the expected URL:
-          <a href="https://identity.ic0.app">https://identity.ic0.app</a>. For
-          security reasons creation of new Internet Identity anchors is disabled
-          on this origin.
-        </p>
-        <p>
-          Please switch to
-          <a href="https://identity.ic0.app">https://identity.ic0.app</a> to
-          register a new Internet Identity anchor.
-        </p>
-        <p>
-          If you were redirected here by another website, please inform the
-          developers. More information is provided
-          <a
-            href="https://forum.dfinity.org/t/internet-identity-proposal-to-deprecate-account-creation-on-all-origins-other-than-https-identity-ic0-app/9760"
-            >here</a
-          >.
-        </p>`,
-    })}
-    <button id="deviceAliasCancel">Cancel</button>
+    <hgroup>
+      <h1 class="t-title t-title--main">
+        Create a new Internet Identity Anchor
+      </h1>
+    </hgroup>
+    <div class="l-stack">
+      ${warnBox({
+        title: "Registration Disabled",
+        message: html`<p class="t-paragraph t-lead">
+            You are <b class="t-strong">not</b> browsing this website on the
+            expected URL:
+            <a class="t-link" href="https://identity.ic0.app">https://identity.ic0.app</a>. For
+            security reasons creation of new Internet Identity anchors is
+            disabled on this origin.
+          </p>
+          <p class="t-paragraph">
+            Please switch to
+            <a class="t-link" href="https://identity.ic0.app">https://identity.ic0.app</a> to
+            register a new Internet Identity anchor.
+          </p>
+          <p class="t-paragraph">
+            If you were redirected here by another website, please inform the
+            developers. More information is provided
+            <a
+              class="t-link"
+              href="https://forum.dfinity.org/t/internet-identity-proposal-to-deprecate-account-creation-on-all-origins-other-than-https-identity-ic0-app/9760"
+              >here</a
+            >.
+          </p>`,
+      })}
+    </div>
+    <div class="l-stack">
```

Comment:
don't know well the css of II but not sure you need that extra `div` for styling purpose

Reply:
Without the `div` the button is too close to the text, even if `l-stack` is set on the button. Not 100% sure why but the `div` doesn't really bother me

Reply:
Can live with it, was just comment, PR is already accepted.

Being said, if I had to bet, I would bet this `div` is not needed.